integration/fuse/bin/alluxio-fuse

```diff
@@ -202,7 +202,7 @@ fuse_stat() {
   local fuse_info="$(ps aux | grep [A]lluxioFuse)"
   if [[ -n ${fuse_info} ]]; then
     echo -e "pid\tmount_point\talluxio_path"
-    echo -e "$(ps ax -o pid,args | grep [A]lluxioFuse | awk -F' ' '{print $1 "\t" $(NF-2) "\t" $NF}')"
+    echo -e "$(ps ax -o pid,args | grep [A]lluxioFuse | awk -F' ' '{print $1 "\t" $(NF-4) "\t" $NF}')"
```

Comment:
Is the behavior of linux and mac same?

Reply:
Could you please try both and paste the results in this PR?

Reply:
The demo is from a linux env. I just tested it with my mac, it is the same.
After applying this PR, it became ok.

```
> integration/fuse/bin/alluxio-fuse mount -o allow_other,default_permissions /tmp/projects /projects
Starting AlluxioFuse process: mounting alluxio path "/projects" to local mount point "/tmp/projects"
usage: sleep seconds
Successfully mounted Alluxio to /tmp/projects.

> jps -m
87837 AlluxioFuse -m /tmp/projects -a /projects -o allow_other,default_permissions

> integration/fuse/bin/alluxio-fuse unmount -f /tmp/projects
Successfully unmount fuse at /tmp/projects

> jps -m
87837 AlluxioFuse -m /tmp/projects -a /projects -o allow_other,default_permissions
```